### PR TITLE
Upload script

### PR DESCRIPTION
**Pull request by jackcarlisle**
Finished the upload bash script tutorial. Ready to merge with master.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,13 @@
   "description": "\"How-to Guide for AWS Lambda, DynamoDB, SNS & API Gateway\"",
   "main": "index.js",
   "scripts": {
+<<<<<<< HEAD
```
Comment (nelsonic) [inline package.json]: @jackcarlisle please remove these merge conflicts from `package.json` ... thanks. :+1:

**Comment by jackcarlisle** [comment]
@nelsonic Sorry I didn't see those! Done

File: README.md [hunk 2]
```diff
@@ -702,6 +703,151 @@ For more information have at the Codeship documentation:
 * [Integrating AWS Lambda with Codeship](https://blog.codeship.com/integrating-aws-lambda-with-codeship/)
 * [Deployment to AWS Lambda](https://codeship.com/documentation/continuous-deployment/deployment-to-aws-lambda/)
 
+<<<<<<< HEAD
```
Comment (nelsonic) [inline README.md]: You were _so_ close...

**Comment by jackcarlisle** [comment]
@nelsonic think that's it!

**Comment by nelsonic** [comment]
Cool. thanks! :+1:

**Pull request merged**